### ISSUE TITLE
plugin boilerplate

### ISSUE THREAD

**Issue by gennaro-tedesco** [rockerBOO/awesome-neovim#65]
I would like to ask if you deem useful to add a program that can provide a plugin boilerplate example (it can be useful for users who want to start out writing their own and browse this page for inspiration). I have written [boilit](https://github.com/gennaro-tedesco/boilit), a simple program that generates a plugin skeleton with files, headers, comments and imports already set-up.

It is not a neovim plugin per sé, so I am hesitant to suggest it: let me know what you think and if you believe it can be of any help we can add it to the list.

Demo: 

https://asciinema.org/a/VpggIG2YeksuuryIHFmVATX43

**Comment by rockerBOO** [comment]
Yes I think this would be good to add. The list should be tailored to neovim (I have a bunch of UI's listed). So we are good there with boilit.

**Comment by gennaro-tedesco** [comment]
I opened https://github.com/rockerBOO/awesome-neovim/pull/86
